src/NServiceBus.AcceptanceTests/Routing/when_replying_to_a_message_sent_to_specific_instance.cs

```diff
@@ -0,0 +1,86 @@
+﻿namespace NServiceBus.AcceptanceTests.ScaleOut
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NServiceBus.Routing;
+    using NUnit.Framework;
+
+    [Explicit("We don't yet have a way to inspect on which receiver a message has been received.")]
```

Comment:
Nope. This is here to remind me that it would be a cool feature to know what is the destination of the received message currently being processed. This is something that makes sense after adding the second receiver but out of scope for this pull.